fastlane/Fastfile

```diff
@@ -48,7 +48,7 @@ platform :ios do
               branch_name: 'ios-beta'
           })
 
-    pilot({distribute_external: true})
+    pilot
```

Comment:
Why did we remove `({distribute_external: true})`? Is that now the default option in the new version of fastlane?